### PR TITLE
Fix the completer in Jupyterlab 4

### PR DESCRIPTION
**Pull request by brichet**
## Describe your changes

This PR fix the SQL completion for jupyterlab 4

## Issue number

Related to https://github.com/ploomber/jupysql/issues/930

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)
- [x] Added [tests](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#testing) (when necessary).
- [x] Added [docstring](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#documenting-changes-and-new-features) documentation and update the [changelog](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#changelog) (when needed)

### CODE REVIEW

File: ui-tests/jupyter_server_test_config.py [hunk 1]
```diff
@@ -4,17 +4,9 @@
...
-c.ServerApp.port = 8888  # noqa: F821
-c.ServerApp.port_retries = 0  # noqa: F821
-c.ServerApp.open_browser = False  # noqa: F821
-
-c.ServerApp.root_dir = mkdtemp(prefix="galata-test-")  # noqa: F821
-c.ServerApp.token = ""  # noqa: F821
-c.ServerApp.password = ""  # noqa: F821
-c.ServerApp.disable_check_xsrf = True  # noqa: F821
-c.LabApp.expose_app_in_browser = True  # noqa: F821
+configure_jupyter_server(c)
```
Comment (edublancas) [inline ui-tests/jupyter_server_test_config.py:9]: is this a new way to configure the server?

we need to add `# noqa: F821` at the end of this line so the lint doesn't complain

File: ui-tests/package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
     "test:update": "jlpm playwright test --update-snapshots"
   },
   "devDependencies": {
-    "@jupyterlab/galata": "^4.3.0",
-    "@playwright/test": "^1.34.2",
+    "@jupyterlab/galata": "^5.0.5",
```
Comment (edublancas) [inline ui-tests/package.json:13]: if upgrading galata breaks things, maybe we can keep the old version? I think it's better to upgrade but if it breaks stuff, we might consider keeping an older version for now

**Review by edublancas — CHANGES_REQUESTED** [review]
I tested it and it works well, just fix the CI. let me know if you need help

File: ui-tests/jupyter_server_test_config.py (hunk 1 above)
Comment (brichet) [reply ui-tests/jupyter_server_test_config.py]: Yes, galata provides the default configuration.

Comment (brichet) [reply ui-tests/jupyter_server_test_config.py]: I added `# noqa: F821`, thanks for the hint

File: ui-tests/package.json (hunk 2 above)
Comment (brichet) [reply ui-tests/package.json:13]: Galata 5 is the version from Jupyterlab 4, I think it's better to upgrade it, there could be some side effect to keep the old version.

**Comment by edublancas** [comment]
awesome! the CI is failing but I merged it anyway since the base `jupyter4` branch is also failing. I'll follow up on the original thread to discuss next steps.

thanks a lot for working on this!

**Pull request merged**